desktop/src/main/java/bisq/desktop/bisq.css

```diff
@@ -867,6 +891,32 @@ textfield */
  * Table                                                                       *
  *                                                                             *
  ******************************************************************************/
+.table-view .table-row-cell:even .table-cell {
+    -fx-background-color: derive(-bs-background-color, 5%);
+    -fx-border-color: derive(-bs-background-color,5%);
```

Comment:
Maybe nitpicky, but there is whitespace inconsistency :)

Reply:
I agree, just run some auto formatter on the code to get spacing uniform. It's a small thing but someone will do it later and there will be changes in an unrelated PR.

Apart from that I'm good to merge this PR.